portal/models/intervention_strategies.py

```diff
@@ -228,21 +227,20 @@ def get_top_level_org_name():
             return top_level_org_name
 
         def get_due_date(assessment_status):
-            #FOR TESTING!!
-            # parent_org = get_top_level_org_name()
-            # thresholds = (
-            #     (90, "CRV"),
-            #     (30, "IRONMAN")
-            # )
-            # for days_allowed, org in thresholds:
-            #     if org == parent_org:
-            #         return assessment_status.consent_date + timedelta(days=days_allowed)
             instrument_due_date = None
-            for instrument, details in assessment_status.instrument_status.items():
-                if not instrument_due_date:
-                    instrument_due_date = details.get('by_date')
+            # prefer due_date for first instrument needing full assessment
+            needing = assessment_status.instruments_needing_full_assessment()
```

Comment:
You might be able to simplify this using `or` in the following way (`or` returns the first value that evaluates to `True`). Not required, just a little easier to read :)
```
instruments = (
    assessment_status.instruments_needing_full_assessment() or
    assessment_status.instruments_in_process()
)
if instruments:
    return assessment_status.instrument_status[instruments[0]].get('by_date')
return datetime.utcnow()
```